web-server-lib/web-server/http/request.rkt

```diff
@@ -602,12 +562,14 @@
                         #:when (bytes=? haystack* needle))
              pos))))
 
-(define (read-mime-multipart in boundary
-                             #:max-files [max-files 100]
-                             #:max-file-length [max-file-length (* 10 1024 1024)]
-                             #:max-file-memory-threshold [max-file-memory-threshold (* 1 1024 1024)]
-                             #:max-fields [max-fields 100]
-                             #:max-field-length [max-field-length (* 8 1024)])
+(define (read-mime-multipart in boundary #:safety-limits [limits (make-safety-limits)])
+  (match-define (safety-limits #:max-request-files max-files
+                               #:max-request-file-length max-file-length
+                               #:request-file-memory-threshold max-file-memory-threshold
+                               #:max-request-headers max-fields
+                               #:max-request-header-length max-field-length)
+    limits)
+  
```

Comment:
I think this change was wrong: `#:max-request-headers` and `#:max-request-header-length` shouldn't control reading the body of `multipart/form-data` parts that end up as `binding:form` values. I will adjust `safety-limits` accordingly.

I think I may have been confused as with `MAX-HEADERS/FIELD` about "field," which obviously refers to form fields, but the phrase "MIME header fields" is also used in RFC 7478 discussing `multipart/form-data`.